dockers/tpu-tests/tpu_test_cases.jsonnet

```diff
@@ -16,7 +16,7 @@ local tputests = base.BaseTest {
   tpuSettings+: {
     softwareVersion: 'pytorch-{PYTORCH_VERSION}',
   },
```

Comment:
Needs to update https://github.com/PyTorchLightning/pytorch-lightning/blame/master/README.md#L91